patches/node/build_use_third_party_simdutf.patch

```diff
@@ -0,0 +1,51 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Mon, 9 Dec 2024 11:18:51 -0600
+Subject: build: use third_party/simdutf
+
+use the Chromium version of simdutf to avoid duplicate symbols
+
+diff --git a/node.gni b/node.gni
+index 99ec540ec41ddf5682eed7618ba87d6935b3b982..d42bef032d2c410f790aae8949d2c2267c58751b 100644
+--- a/node.gni
++++ b/node.gni
+@@ -12,6 +12,8 @@ declare_args() {
+ 
+   node_crypto_path = "//third_party/boringssl"
+ 
++  node_simdutf_path = "//third_party/simdutf:simdutf"
```

Comment:
nit: shorthand if the target matches the path,

```suggestion
+  node_simdutf_path = "//third_party/simdutf"
```

Reply:
Good catch, updated in 3bcc9679bd6bb19515c8932815a993d565feb14